src/components/marketoForms/MarketoForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import { MarketoFormStyleOverwrite, SuccessText } from "./styles";
+
+declare global {
+  interface Window {
+    MktoForms2: {
+      loadForm?: (
+        url: string,
+        munchkinId: string,
+        formid: number,
+        onSubmit?: (form: any) => void
+      ) => void;
+    };
+  }
+}
+
+function appendScript(setScriptLoaded) {
+  if (window.MktoForms2) return setScriptLoaded(true);
+
+  const script = document.createElement("script");
+  script.src = "//info.saasquatch.com/js/forms2/js/forms2.min.js";
+  script.onload = () => (window.MktoForms2 ? setScriptLoaded(true) : null);
+  document.body.appendChild(script);
+}
```

Comment:
This function could be moved inside of the `useMarketo` hook so you don't have to pass `setScriptLoaded` around.